pkg/cmd/apps.go

```diff
@@ -38,6 +39,38 @@ func MakeApps() *cobra.Command {
 		return nil
 	}
 
+	info := &cobra.Command{
+		Use:   "info",
+		Short: "Find info about a Kubernetes app",
+		Long:  "Find info about how to use the installed Kubernetes app",
+		Example: `  k3sup app info [APP]
+  k3sup app info inlets-operator
+  k3sup app info --help`,
+		SilenceUsage: true,
+	}
+
+	info.RunE = func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			fmt.Println("You can get info about: inlets-operator")
+			return nil
+		}
+
+		if len(args) != 1 {
+			fmt.Println("You can get info about exactly one installed app")
```

Comment:
Don't use `fmt.Println` for errors, use `return fmt.Errorf`, this is already handled via `RunE` and the cobra framework.